classes/travel/TravelManager.php

```diff
@@ -266,13 +266,26 @@ public function fetchNearbyPlayers(): array {
                 }
             }
 
-            // only display attack links if the same rank
+            // loot count
+            $loot_count = 0;
+            $loot_result = $this->system->db->query("SELECT COUNT(*) as `count` FROM `loot` WHERE `user_id` = {$user['user_id']} AND `claimed_village_id` IS NULL AND `battle_id` IS NULL LIMIT 1");
+            if ($this->system->db->last_num_rows > 0) {
+                $loot_result = $this->system->db->fetch($loot_result);
+                $loot_count = $loot_result['count'];
+            }
+
+            // only display attack links if the same rank OR carrying loot OR in war action
             $can_attack = false;
-            if ((int)$user['rank'] === $this->user->rank_num
-                && $this->user->location->equals(TravelCoords::fromDbString($user['location']))
+            if ($this->user->location->equals(TravelCoords::fromDbString($user['location']))
                 && $user['user_id'] != $this->user->user_id
                 && $user['village'] !== $this->user->village->name) {
-                $can_attack = true;
+                if ((int)$user['rank'] === $this->user->rank_num) {
+                    $can_attack = true;
+                } else if ($user['operation'] > 0) {
+                    $can_attack = true;
+                } else if ($loot_count > 0) {
+                    $can_attack = true;
+                }
```

Comment:
This gets tricky if we're opening PvP for any war activities, could kinda take away the role of Chuunin - Can you comment this out for now and let's discuss? Would love to get player input on this if we haven't already

Reply:
Will comment out until we discuss the role of Chuunin further.